### PR TITLE
Initial BrowserStack And SauceLabs

### PR DESCRIPTION
**Pull request by Gioserden**
Please someone codereview, thanks.

### CODE REVIEW

**Comment by carledriss** [comment]
@Gioserden 
please solve conflicts against develop

File: gradle.properties [hunk 1]
```diff
@@ -3,11 +3,24 @@
...
+remoteUserName=
+remoteKey=
+
+
+# SAUCELAB AND BROWSERSTACK PROPERTIES
+remoteBrowser=Chrome
+remoteBrowserVersion=58.0
+remotePlatform=Windows
+remotePlatformVersion=8
+remoteResolution=1024x768
```
Comment (carledriss) [inline gradle.properties]: Don't add the environments
add examples as comment

File: src/main/java/org/fundacionjala/pivotal/core/browser/BrowserStack.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+/**
+ * BrowserStack class that implements IBrowsers.
+ */
+public class BrowserStack implements Browser {
+
+  private static final Environment ENV = Environment.getInstance();
+
+  private static final String USER_NAME = ENV.getRemoteUserName();
+  private static final String ACCESS_KEY = ENV.getRemoteKey();
+  private static final String URL = "https://" + USER_NAME + ":" + ACCESS_KEY + "@hub-cloud.browserstack.com/wd/hub";
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/core/browser/BrowserStack.java]: String.format

File: src/main/java/org/fundacionjala/pivotal/core/browser/BrowserStack.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+   *
+   * @return Driver.
+   */
+  @Override
+  public WebDriver getBrowser() {
+    WebDriver driver;
+    try {
+      driver = new RemoteWebDriver(new URL(URL), setCapabilities());
+    } catch (MalformedURLException e) {
+      throw new RuntimeException();
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/core/browser/BrowserStack.java]: It is missing the logger

File: src/main/java/org/fundacionjala/pivotal/core/browser/SauceLabs.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+/**
+ * SauceLabs class that implements IBrowsers.
+ */
+public class SauceLabs implements Browser {
+  private static final Environment ENV = Environment.getInstance();
+
+  private static final String USERNAME = ENV.getRemoteUserName();
+  private static final String ACCESS_KEY = ENV.getRemoteKey();
+  private static final String URL = "https://" + USERNAME + ":" + ACCESS_KEY + "@ondemand.saucelabs.com:443/wd/hub";
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/core/browser/SauceLabs.java]: String.format ?

File: src/test/java/org/fundacionjala/pivotal/runner/Runner.java [hunk 5]
```diff
@@ -18,7 +18,7 @@
                 "html:target/test-report",
                 "json:target/test-report.json",
                 "junit:target/test-report.xml"},
-        features = {"src/test/resources/features"},
+        features = {"src/test/resources/features/ui/workspace"},
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/pivotal/runner/Runner.java]: Do we need to commit this change ?

File: gradle.properties [hunk 6]
```diff
@@ -3,12 +3,27 @@
...
 token=
+
 #CHROME or FIREFOX
-browser=CHROME
+browser=
 dockerurl=
+
 #Fixed
-baseurl=https://www.pivotaltracker.com
+baseurl=
```
Comment (sphinix27) [inline gradle.properties]: Don't erase the url

File: src/main/java/org/fundacionjala/pivotal/core/browser/DockerChromeBrowser.java [hunk 7]
```diff
@@ -32,7 +36,7 @@ public WebDriver getBrowser() {
             driver = new RemoteWebDriver(new URL(Environment.getInstance().getDockerUrl()), chromeCapabilities);
 
         } catch (MalformedURLException e) {
-            //TODO logger.
+            LOGGER.error("Not instance driver");
```
Comment (sphinix27) [inline src/main/java/org/fundacionjala/pivotal/core/browser/DockerChromeBrowser.java]: Please add throw new RuntimeException();

Comment (sphinix27) [reply src/main/java/org/fundacionjala/pivotal/core/browser/DockerChromeBrowser.java]: Add the necessary configuration in build.gradle to run from console

**Pull request merged**